app.py

```diff
@@ -0,0 +1,33 @@
+import os.path
+import tornado.httpserver
+import tornado.ioloop
+import tornado.options
+import tornado.web
+import rethinkdb as db
+from tornado import ioloop, gen
+from tornado.concurrent import Future, chain_future
+from tornado.options import define, options
+
+#
+db.set_loop_type("tornado")
+connection = db.connect(host='localhost', port=28015)
```

Comment:
I was using this interesting [ORM](https://github.com/linkyndy/remodel) to handle all of the boilerplate connection, table creation, etc. Maybe we can connect to the database manually for async or complex queries that the ORM can't handle?